dist/info/beetle_bsnes_libretro.info

```diff
@@ -1,7 +1,7 @@
-display_name = "SNES / Super Famicom (Mednafen bsnes)"
+display_name = "SNES / Super Famicom (Beetle bSNES)"
 authors = "byuu|Ryphecha"
 supported_extensions = "smc|fig|bs|st|sfc"
-corename = "Mednafen/Beetle bsnes"
+corename = "Beetle bSNES"
```

Comment:
https://gitlab.com/higan/higan/blob/72b6a8b32e73c27346f96327e15e15e9e00357f5/higan/sfc/sfc.hpp#L13 says it is not called bSNES.